modules/apps/content-dashboard/content-dashboard-document-library-impl/src/main/resources/META-INF/resources/view_content_dashboard_file_extensions.jsp

```diff
@@ -20,12 +20,9 @@
 ContentDashboardFileExtensionItemSelectorViewDisplayContext contentDashboardFileExtensionItemSelectorViewDisplayContext = (ContentDashboardFileExtensionItemSelectorViewDisplayContext)request.getAttribute(ContentDashboardFileExtensionItemSelectorViewDisplayContext.class.getName());
 %>
 
-<liferay-util:html-top>
-	<link href="<%= PortalUtil.getStaticResourceURL(request, application.getContextPath() + "/css/tree.css") %>" rel="stylesheet" type="text/css" />
-</liferay-util:html-top>
-
 <section class="h-100">
-	<span aria-hidden="true" class="loading-animation mt-0 tree-filter-loader"></span>
+	<span aria-hidden="true" class="loading-animation mt-0 tree-filter-loader"
```

Comment:
I think we can delete that css class `tree-filter-loader` as it wont get load at this point.

I'm moving just this styles inline so we do not have to mess around with the whole CSS file.
The rest of the styles are loaded fine by the JS